src/gatling/scala/config/ApiConfig.scala

```diff
@@ -5,7 +5,7 @@ import com.fasterxml.jackson.annotation.JsonProperty
 class ApiConfig(@JsonProperty("scopes") _scope: String,
                 @JsonProperty("url") _url: String
                ) {
-  val scope = _scope
-  val url = _url
+ lazy val scope = _scope
```

Comment:
Is this still needed to be marked as 'Lazy'?

Reply:
Removed :)